### ISSUE TITLE
Maschine-Readable or plain text to leverage git/github

### ISSUE THREAD

**Issue by rriemann** [Bit-Nation/Pangea-Docs#26]
Hello,

I suggest to leverage the source code and git to allow more easily to change and comment parts of the texts in the Pangea-Docs folder.

I agree that the PDF looks very nice, but it is not very accessible.I would be much in favour of a markdown version (or at least flat ODT (fodt)). With the tool pandoc, you can create all kinds of documents from markdown formatted files.

**Comment by xsttx** [comment]
Yes I agree, we'll add a markdown version, as well. Thanks for the advice re- pandoc, will check it out!
